src/Horse.HTTP.pas

```diff
@@ -136,12 +136,14 @@ procedure THorseRequest.InitializeQuery;
   LItem: string;
   LKey: string;
   LValue: string;
+  p : Integer;  
```

Comment:
please use the default coding style of the horse

Reply:
Try using the name of the local variable with: LEqualFirstPos